cmd/create/cluster/cmd.go

```diff
@@ -2111,15 +2236,24 @@ func run(cmd *cobra.Command, _ []string) {
 			if !output.HasFlag() || r.Reporter.IsTerminal() {
 				r.Reporter.Infof("Preparing to create OIDC Provider.")
 			}
-			oidcprovider.Cmd.Run(oidcprovider.Cmd, []string{clusterName, mode})
+			if oidcEndpointUrl == "" && boundServiceAccountSigningKey == "" {
```

Comment:
NP
Maybe we can extract it to something like `isOidcSet` and then reuse it in line `2254` as well, WDYT?

Reply:
Sure!